CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ This is a list of high-level changes for each release of AWX. A full list of com
 - Fixed a bug that prevented PagerDuty notifications from sending for workflow job template approvals (https://github.com/ansible/awx/issues/7094)
 - Fixed a bug that broke external log aggregation support for URL paths that include the = character (such as the tokens for SumoLogic) (https://github.com/ansible/awx/issues/7139)
 - Fixed a bug that prevented organization admins from removing labels from workflow job templates (https://github.com/ansible/awx/pull/7143)
+- Removed remaining uses of memcache and replaced it with django-redis (https://github.com/ansible/awx/pull/7240) 
```

Comment:
Since we didn't just remove the `python-memcached` library, but also an entire container/dependency of AWX, I think this should go near the top, and should say something more like:

`- Removed memcached as a dependency of AWX (https://github.com/ansible/awx/issues/6932)`

Reply:
:+1: makes sense